lib/galaxy/tool_util/lint.py

```diff
@@ -102,10 +102,11 @@ def name(cls) -> str:
         return cls.__name__
 
     @classmethod
-    def list_listers(cls) -> List[str]:
+    def list_linters(cls) -> List[str]:
         """
         list the names of all linter derived from Linter
         """
+        submodules.import_submodules(galaxy.tool_util.linters)
         return [s.__name__ for s in cls.__subclasses__()]
```

Comment:
Since this was released in packages, we should not break code that uses the previous name:

```suggestion
        return [s.__name__ for s in cls.__subclasses__()]

    list_listers = list_linters  # deprecated alias
```